RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate ReadReceiptView to Typescript

## Proposed changes
The proposal of this PR is migrate `ReadReceiptView` to TypeScript.

## Issue(s)	


## How to test or reproduce

- First, you need to access a server with `Read receipt` activated
- Then you do a long press in any message and tap on `Read Receipt`
- So, the screen should work correctly

## Screenshots

## Types of changes




## Checklist



## Further comments
